app/controllers/offline_reservations_controller.rb

```diff
@@ -8,6 +8,7 @@ class OfflineReservationsController < ApplicationController
   before_action :init_current_facility
   before_action :load_instrument
   before_action :load_reservation, only: %i(edit update)
+  after_action :flag_ongoing_reservations_as_problem, only: %i(create)
```

Comment:
Won't this get called on an unsuccessful `create`? I don't think we'd want that to happen.